app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,44 @@
+class CategoriesController < ApplicationController
+  include CategoriesHelper
```

Comment:
we already spoke about why we should let the view helpers be used only in our views!